stress/common/src/main/java/alluxio/stress/client/ClientIOTaskResult.java

```diff
@@ -262,7 +262,7 @@ public ClientIOSummary aggregate(Iterable<ClientIOTaskResult> results) throws Ex
       for (Map.Entry<Integer, Long> threadCountIOBytesEntry: threadCountIOBytes.entrySet()) {
         int numThreads = threadCountIOBytesEntry.getKey();
         long ioBytes = threadCountIOBytesEntry.getValue();
-        threadCountIoMbps.put(numThreads, (float) ioBytes
+        threadCountIoMbps.put(numThreads, (float) ioBytes / 1000
```

Comment:
are we usually using 1000 or 1024? any constants?

Reply:
I think for `MBps` you should be doing something like this:

```java
public float getIOMBps() {
  return ((float) mIOBytes / getDurationMs()) * 1000.0f / Constants.MB;
}
```

Reply:
Thanks. I messed up the math